cpp/ql/src/semmle/code/cpp/models/implementations/Memset.qll

```diff
@@ -15,7 +15,7 @@ import semmle.code.cpp.models.interfaces.SideEffect
 private class MemsetFunction extends ArrayFunction, DataFlowFunction, AliasFunction,
   SideEffectFunction {
   MemsetFunction() {
-    hasGlobalName(["memset", "wmemset", "bzero", "__builtin_memset", "__builtin_memset_chk"]) or
+    hasGlobalName(["memset", "wmemset", bzero(), "__builtin_memset", "__builtin_memset_chk"]) or
     hasQualifiedName("std", ["memset", "wmemset"])
```

Comment:
Should we refactor this to use `hasGlobalOrStdName`?

Reply:
All the `bzero()` call below do not apply to the C++ `std::bzero`, do they?

Reply:
Is there a `std::bzero`?

Reply:
ah sorry, it is indeed posix-only - disregard my comment then.